pkgs/build-support/build-dotnet-package/default.nix

```diff
@@ -0,0 +1,113 @@
+{ stdenv, lib, makeWrapper, pkgconfig, mono, dotnetbuildhelpers }:
+
+attrsOrig @
+{ baseName
+, version
+, buildInputs ? []
+, placateNuget ? true
+, placatePaket ? true
+, patchFSharpTargets ? true
+, removeDuplicatedDlls ? true
+, xBuildFiles ? [ ]
+, xBuildFlags ? [ "/p:Configuration=Release" ]
+, outputFiles ? [ "bin/Release/*" ]
+, dllFiles ? [ "*.dll" ]
+, exeFiles ? [ "*.exe" ]
+, ... }:
+  let
+    arrayToShell = (a: toString (map (lib.escape (lib.stringToCharacters "\\ ';$`()|<>\t") ) a));
+
+    attrs = {
+      name = "${baseName}-${version}";
+
+      buildInputs = [
+        pkgconfig
+        mono
+        dotnetbuildhelpers
+        makeWrapper
+      ] ++ buildInputs;
+
+      configurePhase = ''
+        runHook preConfigure
+
+        [ -n $placateNuget ] && placate-nuget.sh
+        [ -n $placatePaket ] && placate-paket.sh
+        [ -n $patchFSharpTargets ] && patch-fsharp-targets.sh
```

Comment:
patch-fsharp-targets.sh puts in the variable $(FSharpTargets...) in the build files. The variable as in the reference, not its value.